cadc-vos-server/src/main/java/ca/nrc/cadc/vos/server/transfers/PullFromVOSpaceNegotiation.java

```diff
@@ -98,31 +102,75 @@
     private static final Logger log = Logger.getLogger(PullFromVOSpaceNegotiation.class);
 
     private VOSpaceAuthorizer authorizer;
+    private boolean isPackageViewTransfer = false;
 
     public PullFromVOSpaceNegotiation(NodePersistence per, JobUpdater ju, Job job, Transfer transfer) {
         super(per, ju, job, transfer);
         this.authorizer = new VOSpaceAuthorizer(true);
         authorizer.setNodePersistence(nodePersistence);
     }
 
+    @Override
+    public void validateView() throws IllegalArgumentException,
+            TransferException, Exception {
+        // There is no representation for the package view in View.properties,
+        // so this function needs to do some extra work for this instance.
+        // For all other View values, the standard validation is performed.
+
+        if (TransferUtil.isPackageTransfer(transfer)) {
+            isPackageViewTransfer = true;
+        } else {
+            super.validateView();
+        }
+    }
+
     @Override
     public void doAction()
         throws TransferException, JobPersistenceException, JobNotFoundException,
         LinkingException, NodeNotFoundException, TransferParsingException,
         IOException, TransientException, URISyntaxException {
         boolean updated = false;
         try {
+
+            if (isPackageViewTransfer) {
+                // Set job back to PENDING so next service in the chain can pick it up
+                // (in this case /vault/pkg/<jobid>
+                updateTransferJob(null, null, ExecutionPhase.PENDING);
+
+                // Add responseFormat parameter to the job, using the parameter
+                // type from the view
+                Parameter p = TransferUtil.viewParam2JobParam(VOS.PROPERTY_URI_FORMAT,
+                    "responseFormat", transfer);
```

Comment:
You should check with PackageRunner, but I think the param is in all caps:  RESPONSEFORMAT

Reply:
changed to RESPONSEFORMAT